mne_bids/mne_bids.py

```diff
@@ -39,6 +39,8 @@
                  '.fif': 'Elekta', '.gz': 'Elekta', '.pdf': '4D Magnes',
                  '.ds': 'CTF'}
 
+IGNORED_CHANNELS = ['STI 014']
```

Comment:
I would add a condition within _channels_tsv using the helper functions that identifies the extension and manufacturer and say if it is KIT, ignore 'STI 014'

Reply:
sounds reasonable. Will make a fix and add to this commit